test/district_repository_test.rb

```diff
@@ -111,4 +117,17 @@ def test_returns_an_empty_array_if_fragment_doesnt_match_a_district
     assert_equal 0, d.count
     assert_equal [], d
   end
+  
+  def test_creates_a_relationship_between_district_and_enrollment
+    dr = DistrictRepository.new
+    dr.load_data({
+      enrollment: {
+        kindergarten: "./test/sample_kindergarten.csv"
+      }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.